components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -131,6 +131,7 @@ class PromptFeature private constructor(
     private val shareDelegate: ShareDelegate,
     override val loginValidationDelegate: LoginValidationDelegate? = null,
     private val isSaveLoginEnabled: () -> Boolean = { false },
+    private val isCCAutofillEnabled: () -> Boolean = { false },
```

Comment:
I think we should prefer saying CreditCard instead of CC. Can you also add the @property?

```suggestion
    private val isCreditCardAutofillEnabled: () -> Boolean = { false },
```

Reply:
Sure, thank you!